index.js

```diff
@@ -1,16 +1,27 @@
 var express = require('express');
-var app = express();
-var port = process.env.PORT || 1347;
+var winston = require('winston');
+var bodyParser = require('body-parser');
 
-app.get('/', function (req, res) {
-  res.send('Hello World!');
-});
+var app = module.exports = express();
 
-var server = app.listen(port, function () {
+winston.level = 'debug';
+
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json());
 
-  var host = server.address().address;
-  var port = server.address().port;
+app.get('/', function (req, res) {
+  winston.info("GET / request");
+  winston.log('debug', req.body);
+  res.send("hi");
+});
 
-  console.log('Example app listening at http://%s:%s', host, port);
+app.post('/repos/:owner/:repo/hooks/:id/tests', function (req, res) {
+  winston.info("POST /repos/:owner/:repo/hooks/:id/tests request");
+  winston.log('debug', req.body);
+  res.send(req.body);
+});
 
+var port = process.env.PORT || 3002;
+var server = app.listen(port, function () {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['server' is defined but never used.](https://www.codacy.com/app/jeffhann/git-mirror-sync/file/750685227/issues/source?bid=2446500&fileBranchId=2446713#l25)